src/legacy/ui/public/agg_types/utils.tsx

```diff
@@ -17,6 +17,9 @@
  * under the License.
  */
 
+import { parseInterval } from '../utils/parse_interval';
+import { leastCommonInterval } from '../vis/lib/least_common_interval';
+
 function isValidJson(value: string): boolean {
```

Comment:
I know you didn't write this function in this PR, but this doesn't seem right at all. Either the logic is wrong, or the name is. For example: `isValidJson('')` should be false, as such a value would cause JSON.parse to fail. It seems worth putting a doc comment on here to indicate that it will return true for invalid JSON in some edge cases or something along those lines..

Reply:
I added a comment